docs/lang/cs/ANDROID.md

```diff
@@ -0,0 +1,41 @@
+| Aktualizováno 26.02.2023 | Jazyky: CZ, [EN](/docs/ANDROID.md) |
```

Comment:
I suggest to make this date match the date of the original document that you translated.

Reply:
Wasn't in original file so...

07.02 from last commit on file is good?